platform/measurement_utils.cpp

```diff
@@ -18,15 +18,21 @@
 
 namespace measurement_utils
 {
-std::string ToStringPrecision(double d, int pr)
+// Global Locale variable to store system locale.
+static platform::Locale g_systemLocale = {
```

Comment:
How will it work on iOS?

Reply:
On iOS we already confirmed that the app is re-started when the locale is changed, so we decided [above](https://github.com/organicmaps/organicmaps/pull/7779#discussion_r1566352616) to remove locale change detection in iOS.
On iOS, the locale is set in cpp only at app start-up.